heudiconv/heuristics/reproin.py

```diff
@@ -523,6 +546,19 @@ def infotodict(
                 # label for dwi as well
                 datatype_suffix = "dwi"
 
+            # Add "part" entity as needed
+            if datatype != "fmap" and not curr_seqinfo.series_description.endswith(
+                "_SBRef"
+            ):
+                if "P" in curr_seqinfo.image_type:
+                    series_info["part"] = "phase"
```

Comment:
codecov says that this line is not covered, not yet sure how wide implication is ;-)